callbacks/control_bar.py

```diff
@@ -183,6 +192,8 @@ def annotation_mode(
     if generate_modal_opened or any(edit_modal_opened):
         # user is going to type on this page (on a modal) and we don't want to trigger this callback using keys
         raise PreventUpdate
+    if control_accordion_state is not None and "run-model" in control_accordion_state:
```

Comment:
Disabling key binds while interacting with the model configuration tab makes a lot of sense. While testing this at my end, I accidentally left the model configuration tab open while meaning to do some more annotations. Here I noticed that the key binds were off because the model configuration tab was still open.
My suggestion is to maybe allow only 1 tab/accordion item in the left panel to be active at the time. So when the user selects one, the other close automatically (between model and annotations tabs).

Reply:
I think I would prefer classes to be visible while interacting with the result overlay. Perhaps we want to explore splitting out job submission and/or results into an additional tab?